Formula/glib.rb

```diff
@@ -62,6 +63,21 @@ def install
               "giomoduledir=#{HOMEBREW_PREFIX}/lib/gio/modules",
               "giomoduledir=${libdir}/gio/modules"
 
+    # Delete python files because they are provided by `glib-utils`
+    dummy_python_shebang_rewrite_info = python_shebang_rewrite_info("#{HOMEBREW_PREFIX}/bin/python3")
+    prefix.glob("**/*").each do |f|
+      next unless f.file?
+
+      if f.to_s.end_with?(".py")
+        f.unlink
+      elsif dummy_python_shebang_rewrite_info.regex.match?(f.read(dummy_python_shebang_rewrite_info.max_length))
```

Comment:
Why not just include this condition above so that we don't need two branches that do the same thing?

Reply:
> Why not just include this condition above?

`brew style` complains line-too-long.

```bash
$ brew style --fix Formula/glib.rb 
Formula/glib.rb:71:119: C: Line is too long. [131/118]
      if f.extname == ".py" || dummy_python_shebang_rewrite_info.regex.match?(f.read(dummy_python_shebang_rewrite_info.max_length))
                                                                                                                      ^^^^^^^^^^^^^

1 file inspected, 1 offense detected
```

Reply:
Your solution from `glib-utils` using `next if...` twice works too. But we really should not be using the implementation details of `python_shebang_rewrite_info` here.